lib/data_aggregator/darwin_core/publication/eml_file.ex

```diff
@@ -130,6 +130,8 @@ defmodule DataAggregator.DarwinCore.Publication.EmlFile do
     do:
       "This work is licensed under a <ulink url=\"https://creativecommons.org/licenses/by-nc/4.0/legalcode\"><citetitle>Creative Commons Attribution (CC-BY-NC) 4.0 License</citetitle></ulink>. "
 
+  defp intellectual_rights(_), do: ""
```

Comment:
fallback/default case to not receive a missing pattern match error in case there is no license information